### ISSUE TITLE
[Question] Moving CM from btrfs to btrfs not possible?

### ISSUE THREAD

**Issue by Lennolium** [007revad/Synology_app_mover#127]
Hello,

First, thanks for your great work! Really appreciated.

I'm trying to move my Container Manager/docker from volume1 (btrfs HDD SHR1) to volume2 (btrfs 2x Nvme SSDs SHR1) but if I select
the corresponding menu entry the following message appears and the script exits:

`WARNING Do not move ContainerManager from btrfs volume to btrfs volume!`

Is it really not possible to move docker in that way or am I missing something. I searched the issue tab, but did not find anything related. I am graceful for any help.

The full output:
```Synology_app_mover v4.0.71
DS918+ DSM 7.2.2-72806 

Running from: /volume1/homes/Lennart/SynologyAppMover/Synology_app_mover-4.0.71/syno_app_mover.sh

1) Move
2) Backup
3) Restore
Select the mode: 1
You selected Move

[Installed package list]
 1) /volume1  Active Backup for Business
 2) /volume1  Advanced Media Extensions
 3) /volume1  Antivirus Essential
 4) /volume1  Cloud Sync
 5) /volume1  Container Manager
 6) /volume1  Document Viewer
 7) /volume1  exFAT Access
 8) /volume1  Git
 9) /volume1  Hyper Backup
10) /volume1  MariaDB 10
11) /volume1  Node.js v14
12) /volume1  Node.js v16
13) /volume1  Node.js v18
14) /volume1  Node.js v20
15) /volume1  Perl
16) /volume1  PHP 7.3
17) /volume1  PHP 7.4
18) /volume1  PHP 8.0
19) /volume1  PHP 8.2
20) /volume1  Plex Media Server
21) /volume1  Snapshot Replication
22) /volume1  Storage Analyzer
23) /volume1  SynoCli File Tools
24) /volume1  SynoCli Kernel Tools
25) /volume1  SynoCli Monitor Tools
26) /volume1  SynoKernel USB Serial drivers
27) /volume1  Synology Application Service
28) /volume1  Synology Calendar
29) /volume1  Synology Contacts
30) /volume1  Synology Drive Server
31) /volume1  Synology Mail Server
32) /volume1  Synology Office
33) /volume1  Synology Photos
34) /volume1  Text Editor
35) /volume1  Universal Viewer
36) /volume1  VPN Server
37) /volume1  WebDAV Server
38) /volume1  WireGuard
39) /volume2  Virtual Machine Manager
Select the package to move: 5
You selected Container Manager in /volume1

Destination volume is /volume2

WARNING Do not move ContainerManager from btrfs volume to btrfs volume!```

**Comment by 007revad** [comment]
`WARNING Do not move ContainerManager from btrfs volume to btrfs volume!`

Someone else reported getting `WARNING Do not move ContainerManager from ext4 volume to ext4 volume!`

I'll have a look at the script and upload a fixed version shortly.

**Comment by Lennolium** [comment]
Thanks a lot! Don't stress, take your time :)

> `WARNING Do not move ContainerManager from btrfs volume to btrfs volume!`
> 
> Someone else reported getting `WARNING Do not move ContainerManager from ext4 volume to ext4 volume!`
> 
> I'll have a look at the script and upload a fixed version shortly.

**Comment by 007revad** [comment]
All fixed in v4.0.72 
https://github.com/007revad/Synology_app_mover/releases

Is was hard to find the cause which was a non-printable character.
